examples/ios/swift-2.2/GettingStarted.playground/Contents.swift

```diff
@@ -63,9 +67,15 @@ try! realm.write {
 
 let favorites = ["Jennifer"]
 
+#if swift(>=3.0)
+let favoritePeopleWithSpousesAndCars = realm.allObjects(ofType: Person.self)
+    .filter(using: "cars.@count > 1 && spouse != nil && name IN %@", favorites)
+    .sorted(onProperty: "age")
```

Comment:
I had a PR to fix some of the most egregious regressions but it got shunted aside. We can revisit it, or just wait until the GM.